services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/S3TransferManager.java

```diff
@@ -157,6 +157,9 @@
  *         // Wait for the transfer to complete
  *         CompletedCopy completedCopy = copy.completionFuture().join();
  * }
+ * <b> Warns the user if a multi-part operation disabled client is provided for TransferManager to use
```

Comment:
This paragraph is a bit confusing. Suggesting something like below:
```
The automatic parallel transfer feature (multipart upload/download) is available 
through the AWS-CRT based S3 client {@link S3AsyncClient.crtBuilder().build)}  
and Java-based S3 multipart client {@link S3AsyncClient.builder().multipartEnabled(true).build()}. 
If no client is configured, AWS-CRT based S3 client will be used if AWS CRT is in the classpath,
otherwise, Java-based S3 multipart client will be used.
```

Can we also add code snippet for how to construct S3 multipart client in the top?